runtime/browser/xwalk_browser_context.cc

```diff
@@ -349,6 +315,17 @@ net::URLRequestContextGetter*
 #endif
 }
 
+net::URLRequestContextGetter* XWalkBrowserContext::CreateMediaRequestContext() {
```

Comment:
Let's narrow down our problem area. We have 2 new functions(I'm not considering, GetMediaRequestContextForRenderProcess() as it's totally been removed from everywhere),
1. CreateMediaRequestContext()
2. CreateMediaRequestContextForStoragePartition()
 which seem like are new versions of 
a. GetMediaRequestContext()
b. GetMediaRequestContextForStoragePartition()
Out of that 'a' was returning just GetRequestContext() anyway so it makes sense to have been replaced by url_request_getter_.get().
Regarding 2 replacing b I see we are losing some logic there but if you look at shell_browser_context, CastBrowserContext, HeadlessBrowserContext, AwBrowserContext.. are all returning null_ptr except ProfileImpl and OffTheRecordProfileImp so they are returning nullptr if it is android or not. I am not sure if we should follow ProfileImpl or OffTheRecordImpl. There could be a potential issue with 2 but let's not spend too much time on what could be in there from Crosswalk point of view since neither of us know what should go in there. If something breaks we can always fix it later.